corehq/apps/tour/models.py

```diff
@@ -0,0 +1,34 @@
+import datetime
+from django.contrib.auth.models import User
+from django.db import models
+import json_field
+
+
+class GuidedTours(models.Model):
+    user = models.ForeignKey(
+        User,
+        on_delete=models.PROTECT,
```

Comment:
I think based on my reading of [the docs](https://docs.djangoproject.com/en/1.9/ref/models/fields/) that this means that every time you try to delete a `User` it will succeed if they have no tours defined, but will fail if they have tours defined. is that your intention? i would have thought that `CASCADE` would be more appropriate as I don't see why this model should prevent `User`s from being deleted